android/src/main/java/com/stripeterminalreactnative/Mappers.kt

```diff
@@ -659,7 +602,7 @@ private fun mapFromWechatPayDetails(wechatPayDetails: WechatPayDetails?): Readab
 private fun mapFromOfflineDetails(offlineDetails: OfflineDetails?): ReadableMap? =
     offlineDetails?.let {
         nativeMapOf {
-            putString("storedAt", offlineDetails.storedAt.toString())
+            putString("storedAt", offlineDetails.storedAtMs.toString())
```

Comment:
Does it make sense to update the string name? 

```suggestion
            putString("storedAtMs", offlineDetails.storedAtMs.toString())
```

Reply:
Looks like it's still `storedAt` [for iOS](https://stripe.sourcegraphcloud.com/stripe-ios/stripe-terminal-ios/-/blob/StripeTerminal/SCPOfflineDetails+Private.h?L24). @bric-stripe this could be something to consider for SDKv5? 

[Android Jira ticket ](https://jira.corp.stripe.com/browse/TERMINAL-40461)for context.

Reply:
it makes sense for the RN layer to expose a consistent value though, right? I think RN exposing `storedAtMs` and then in the swift layer below it can just convert from the s to the ms value so its consistent regardless of platform...and yeah we should fix it up on the native side. we could add the storedAtMs next to storedAt in a minor update and then drop storedAt in v5

Reply:
Just found RN already had the transform before 4.0, so we can align the name as bric suggested.

https://github.com/stripe/stripe-terminal-react-native/blob/main/ios/Mappers.swift#L530
https://github.com/stripe/stripe-terminal-react-native/blob/main/ios/Mappers.swift#L457-L463

Reply:
updated